en-US/firefox/facebookcontainer/index.lang

```diff
@@ -0,0 +1,74 @@
+## NOTE: Demo page at https://www-dev.allizom.org/firefox/facebookcontainer/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/facebookcontainer/
+
+
+# HTML page title
+;Facebook Container for Firefox | Prevent Facebook from seeing what websites you visit.
+Facebook Container for Firefox | Prevent Facebook from seeing what websites you visit.
+
+
+# HTML page description
+;Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+
+
+;Facebook. Well contained. Keep the rest of your life to yourself.
+Facebook. Well contained. Keep the rest of your life to yourself.
+
+
+;Download Firefox and get the Facebook Container Extension
+Download Firefox and get the Facebook Container Extension
+
+
+# The link points to https://addons.mozilla.org/firefox/addon/facebook-container/
+;Get the Facebook Container Extension
+Get the Facebook Container Extension
+
+
+;The Facebook Container Extension is not available on mobile devices.
+The Facebook Container Extension is not available on mobile devices.
+
+
+# GLink points to https://www.mozilla.org/firefox/focus/; Product name in German is Firefox Klar.
+;Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+
+
+;Download on the App Store
+Download on the App Store
```

Comment:
We already have this string in a shared file, that's why I didn't include in my comment.

Having said that, your call, you can keep it or remove it.